recipe/charts_bld.bat

```diff
@@ -0,0 +1,13 @@
+pushd pyqt_charts
```

Comment:
Any reason to change the name of the scripts used before? I mean, before we had (for instance) `bld-pyqtcharts.bat` and you changed that to `charts_bld.bat`, which makes reviewing your changes harder.

If it's not too much to ask, could you revert to the previous naming? That would also make easier to follow the git history.

Reply:
The build system used to compile PyQt is completely different from the one used before, so the diff will be deleting and adding new lines. I can rename them, but for reviewing purposes, it won't make much difference